src/Spd.Manager.Licence/BizProfileValidation.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+
+namespace Spd.Manager.Licence;
+public class BizProfileUpdateRequestValidator : AbstractValidator<BizProfileUpdateRequest>
+{
+    public BizProfileUpdateRequestValidator()
+    {
+        RuleFor(r => r.BizTradeName).NotEmpty();
+        RuleFor(r => r.BizTypeCode).NotEmpty().IsInEnum();
+        RuleFor(r => r.BizBCAddress).NotEmpty();
```

Comment:
it is not the rule, the rule is if BizAddress is not in BC, then BizBCAddress must exist

Reply:
Fixed.